### PR TITLE
Hotfix: Missing function "sanitize_package"

### PR DESCRIPTION
**Pull request by ScreamingDev**
As reported by a user PHP throws an error
and complains about a missing method.
Instead of removing the wrong code the method has been introduced here.
This means duplicate code but fixes another
number formatting problem at the moment.

What has changed?

- Introduced missing method.

## Things to review

Common things:

- [x] Works fine.
- [x] Changelog written or not needed.
- [ ] In case of bugfix also merged in other minor versions.
- [x] No conflict with current branch.

Got an idea while looking at the GUI?
Please [add a new issue refering to this one.](https://github.com/awsmug/shipcloud-for-woocommerce/issues/new?title=Idea&body=While%20looking%20at%20issue%20)

### CODE REVIEW

File: components/woo/order-bulk.php [hunk 1]
```diff
@@ -430,6 +430,28 @@ protected function create_label_for_order( $order_id, $request ) {
...
+	 * - Have local decimal separator.
+	 *
+	 * @since 1.5.1
+	 *
+	 * @param array $package_data
+	 *
+	 * @return array
+	 */
+	protected function sanitize_package( $package_data ) {
+		$package_data['width']  = wc_format_decimal( $package_data['width'] );
```
Comment (andrecedik) [inline components/woo/order-bulk.php]: I'm getting the following error while trying to create a bulk shipment:
```
Uncaught Error: Cannot use object of type ShipcloudDomainPackage as array in /kunden/440816_22081/webseiten/woocommerce-shipcloud-dev/wordpress/wp-content/plugins/shipcloud-for-woocommerce/components/woo/order-bulk.php:446
```

Comment (ScreamingDev) [reply components/woo/order-bulk.php]: Interesting. Thanks to that I found a complete different approach.

**Comment by andrecedik** [comment]
Is there a quick way you could check this? I'm approving the changes for now, though.

**Comment by andrecedik** [comment]
Works 👍

**Pull request merged**